compiler/src/iree/compiler/Codegen/LLVMCPU/test/aarch64_vector_lowering.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: iree-opt %s --pass-pipeline="builtin.module(func.func(iree-llvmcpu-mmt4d-vector-lowering),iree-codegen-llvmcpu-vector-lowering-pipeline)" --split-input-file | FileCheck %s
+// RUN: iree-opt %s --pass-pipeline="builtin.module(func.func(iree-llvmcpu-mmt4d-vector-lowering),iree-codegen-llvmcpu-vector-lowering-pipeline)" --split-input-file | FileCheck %s -check-prefix=CHECK-KERNEL
```

Comment:
I think you need to add `--iree-llvmcpu-enable-vector-contract-custom-kernels` to the new run

Reply:
I restructured the tests with a positive and negative case. This verifies the `vector-contract-custom-kernels` option enables and disables the custom kernels by itself.